kill-microservice/README.md

```diff
@@ -0,0 +1,90 @@
+
+[chaostoolkit]: https://chaostoolkit.org/
+[docs]: https://docs.chaostoolkit.org
+
+### Install Any Required Drivers
+
+The chaostoolkit is extended by adding [drivers] This experiment requires the following drivers:
+
+* Kubernetes Chaos Toolkit Driver [kubernetes-driver] [kubernetes-driver-github]
+
+[drivers]: https://docs.chaostoolkit.org/drivers/overview/
+[kubernetes-driver]: https://docs.chaostoolkit.org/drivers/kubernetes/
+[kubernetes-driver-github]: https://github.com/chaostoolkit/chaostoolkit-kubernetes
+
+## Running the Experiment
+
+
+### Required Parameters
+
+This experiment requires the following parameters:
+
+* Environment Variables
+  * `APPLICATION_ENTRYPOINT_URL` - Specifies the application entry point URL where your application can be reached in your environment.
+  *  `SERVICE_NAME` - the name of the service that you are going to kill
+
+## Running the Experiment Direct from the Catalog using HTTP
+
+You can run this experiment using the native `chaos` command, or using
+`docker`.
+
+With the native `chaos` command:
+
+```bash
+(chaostk) $ export APPLICATION_ENTRYPOINT_URL=http://192.168.99.100:32474; \
+            export SERVICE_NAME=my-service; \
+                   chaos run https://raw.githubusercontent.com/open-chaos/experiment-catalog/kill-microservice/kill-microservice/kill-microservice.json
+```
+
+> ***NOTE:*** The APPLICATION_ENTRYPOINT_URL & SERVICE_NAME should be setup according to your Kubernetes run time environment. The docker command also uses the .chaostoolkit/settings.yaml from the users home directory.
+
+With `docker`:
+
+```bash
+$ docker run -it \
+           -e APPLICATION_ENTRYPOINT_URL=http://192.168.99.100:32474 \
+           -e SERVICE_NAME=my-service \
+           -v `pwd`:/tmp/result \
+           -v ~/.chaostoolkit:/tmp/settings \
+           quay.io/chaosiq/chaostoolkit-cloud:0.1 \
+           --settings /tmp/settings/settings.yaml \
+           run https://raw.githubusercontent.com/open-chaos/experiment-catalog/kill-microservice/kill-microservice/kill-microservice.json
+```
+
+
+### Running the Experiment from a Local Copy
+
+You can run this experiment using the native `chaos` command, or using
+`docker`.
+
+With the native `chaos` command:
+
+```bash
+(chaostk) $ export APPLICATION_ENTRYPOINT_URL=http://192.168.99.100:32474; \
+            export SERVICE_NAME=my-service; \
+            chaos run kill-pods-experiment.json
+```
+
+> ***NOTE:*** The APPLICATION_ENTRYPOINT_URL, POD_LABEL & POD_NAMESPACE should be setup according to your Kubernetes run time environment. The docker command also uses the .chaostoolkit/settings.yaml from the users home directory.
+
+With `docker`:
+
+```bash
+$ docker run -it \
+           -e APPLICATION_ENTRYPOINT_URL=http://192.168.99.100:32474 \
+           -e SERVICE_NAME=my-service \
+           -v `pwd`:/tmp/result \
+           -v ~/.chaostoolkit:/tmp/settings \
+           quay.io/chaosiq/chaostoolkit-cloud:0.1 \
```

Comment:
See comment from before on the docker image to probably prefer to use: https://github.com/open-chaos/experiment-catalog/blob/master/file-must-not-disappear/README.md#running-the-experiment-from-a-local-copy